l10n/en-US/viewer.properties

```diff
@@ -101,6 +101,7 @@ print_progress_close=Cancel
 # (the _label strings are alt text for the buttons, the .title strings are
 # tooltips)
 toggle_sidebar.title=Toggle Sidebar
+toggle_sidebar_notification.title=Toggle Sidebar (document contains outline/attachment)
```

Comment:
D'oh; would you believe me if I said that I always intended to use the plural form (i.e. `attachments`), and even *thought* that I actually did :-)
Somehow though, when writing this the `s` went missing, probably because `outline` should be in singular.
@timvandermeij Very good catch, I'll address this once you've had time to finish the review (no rush though)!